geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateJndiBindingCommand.java

```diff
@@ -100,7 +100,7 @@
   static final String XA_DATASOURCE_CLASS__HELP =
       "The fully qualified name of the javax.sql.XADataSource implementation class.";
   static final String IFNOTEXISTS__HELP =
-      "Skip the create operation when a Jndi binding with the same name already exists. The default is to overwrite the entry (false).";
+      "Skip the create operation when a jndi binding with the same name already exists. The default is not to skip it.";
```

Comment:
I'd phrase it as
```
"Skip the create operation when a jndi binding with the same name already exists.  Without this option, this action throws an error."
```

Reply:
Changed it to be -> "Skip the create operation when a jndi binding with the same name already exists.  Without specifying this option, this command execution results into an error."